src/router/routes.ts

```diff
@@ -44,8 +44,38 @@ export const routes = [
     }
   },
   {
-    path: '/restoration',
-    name: RouteNames.RESTORATION,
+    path: '/limitedRestorationToFull',
+    name: RouteNames.RESTORATION_CONVERSION,
+    component: Restoration,
+    meta: {
+      requiresAuth: true,
+      isStaffOnly: true,
+      filingType: FilingTypes.RESTORATION
+    }
+  },
```

Comment:
For backwards compatibility, all the new endpoints point to the same component.  Once this PR is approved, we can point the endpoints to a dedicated view.